multisig/simulations/opbnbtestnet/vip-018/simulations.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from "chai";
+import { BigNumber, Contract, utils } from "ethers";
+import { ethers } from "hardhat";
+
+import { calculateMappingStorageSlot } from "../../../../src/utils";
+import { forking, pretendExecutingVip } from "../../../../src/vip-framework";
+import vip018 from "../../../proposals/opbnbtestnet/vip-018";
+import XVSVault_ABI from "./abi/XVSVault_ABI.json";
+
+const XVS_VAULT_PROXY = "0xe4DE4460c93Af011DC0624bc7Ce4a2Bf40feE4ea";
```

Comment:
This address belongs to XVS_VAULT_IMPLEMENTATION. XVS_VAULT_PROXY is at 
`0xB14A0e72C5C202139F78963C9e89252c1ad16f01`